File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        answer = []
        for i in range(0, len(nums)):
            for j in range(0, len(nums)):
                if nums[i] + nums[j] == target and i != j:
                    answer.append(i)
                    answer.append(j)
                    answer = sorted(answer)
                    return answer

        

File: Data Structures & Algorithms/binary-search/submission-1.cpp
class Solution {
public:
    int search(vector<int>& nums, int target) {
       int n = nums.size();
       int x = 0;
       int y = nums.size();
       int z = (x + y)/2;
       while(n>0){
        if (nums[z] == target){
            return z;
        }
        else if (nums[z] < target){
            n = n/2;
            x = z;
            z = (x + y)/2;
        }
        else{
             y = z;
             n = n/2;
             z = (x + y)/2;
        }
       }
       return -1;
    }
};
